### PR TITLE
fix(StatusLetterIdentIcon): Display emoji as svg

### PR DESCRIPTION
**Pull request by igor-sirotin**
Required by status-im/status-desktop#5554

image

image

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Components/StatusLetterIdenticon.qml [hunk 1]
```diff
@@ -12,15 +12,32 @@ Rectangle {
...
     property int letterSize: 21
     property int charactersLen: 1
+    readonly property string emojiId: Emoji.iconId(emoji, emojiSize) || ""
 
     color: Theme.palette.miscColor5
     width: 40
     height: 40
     radius: width / 2
 
+    Image {
```
Comment (osmaczko) [inline src/StatusQ/Components/StatusLetterIdenticon.qml:22]: (optional) Would be great if you move `SVGImage` to StatusQ and use it here.

**Review by osmaczko — APPROVED** [review]

File: src/StatusQ/Components/StatusLetterIdenticon.qml (hunk 1 above)
Comment (osmaczko) [reply src/StatusQ/Components/StatusLetterIdenticon.qml:22]: Sounds good, go for it :+1:

**Review by osmaczko — APPROVED** [review]
:+1:

File: src/StatusQ/Components/StatusEmoji.qml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    
+    width: 14
+    height: 14
+    sourceSize.width: width
+    sourceSize.height: height
+
+    fillMode: Image.PreserveAspectFit
+    mipmap: true
+    antialiasing: true
+    source: emojiId ? `../../assets/twemoji/svg/${emojiId}.svg` : ""
```
[the remark this answers is not in the document]
Comment (osmaczko) [reply src/StatusQ/Components/StatusEmoji.qml:18]: Hm, according to [this](https://stackoverflow.com/a/41321002) it should work. Ignore this comment, I can see you marked "test changes in [status-desktop](https://github.com/status-im/status-desktop)".

**Review by alexandraB99 — APPROVED** [review]
perfect!

**Pull request merged**